mailscanner/msre_edit.php

```diff
@@ -291,112 +257,115 @@ function Show_Form($status_msg, $short_filename, $file_contents, $CONF_ruleset_k
             $desc_field = 'rule' . $rule_count . '_description';
             $hidden_field_code = '';
         }
-        $rule_action_select_html = "<select name=\"$rule_action_select\"";
-        // if this is the default rule, the select box is disabled,
+        $rule_action_select_html = '<select name="' . $rule_action_select . '"';
+        
+        // If this is the default rule, the select box is disabled,
         // because you can't delete, disable, or enable the default
         // rule, only change it.  Originally I had it not there at
         // all, but w/the new way i'm writing the rules (w/the border),
         // each one is in a seperate table, and they don't line up
-        // w/out the select box...
+        // w/out the select box.
         if (strtolower($rule_part['1target'] === 'default')) {
             $rule_action_select_html .= ' disabled';
         }
-        // now continue on..
-        $rule_action_select_html .= ">\n" .
-            "  <option value=\"\" selected>----</option>\n" .
-            "  <option value=\"Delete\">Delete</option>\n" .
-            $rule_action_select_options . "</select>\n";
+        
+        // Now continue on.
+        $rule_action_select_html .= '>' . "\n" .
+            '  <option value="" selected>----</option>' . "\n" .
+            '  <option value="Delete">' . __('delete55') . '</option>' . "\n" .
+            $rule_action_select_options . '</select>' . "\n";
         $desc_text = array(
             $rule_action_select_html => 'rowspan="3"',
-            '<b>Description:</b>&nbsp;&nbsp;<input type="text" ' .
-            "name=\"$desc_field\" size=\"95\" value=\"" . $desc_value . '"' .
+            '<b>' . __('description55') . '</b>&nbsp;&nbsp;<input type="text" ' .
+            'name="' . $desc_field . '" size="95" value="' . $desc_value . '"' .
             $disable_desc_text . '>' . $hidden_field_code
             => 'colspan="' . (MSRE_COLUMNS - 1) . '"'
         );
 
         foreach ($rule_part as $key => $value) {
             $part_name = 'rule' . $rule_count . '_' . preg_replace("/\d/", '', $key);
-            // depending on what part it is, we may need to do something
-            // special, like create a select or checkbox
+            // Depending on what part it is, we may need to do something
+            // special, like create a select or checkbox.
             switch (strtolower($key)) {
                 case '2and':
-                    // and gets a checkbox
+                    // And gets a checkbox
                     $field_name = $part_name;
                     if ($rule_disabled) {
                         $field_name .= '_disabled';
                     }
-                    $checkbox_html = "<input type=\"checkbox\" name=\"$field_name\" value=\"";
-                    $checkbox_html .= 'and';
+                    $checkbox_html = '<input type="checkbox" name="' . $field_name . '" value="';
+                    $checkbox_html .= __('and55');
```

Comment:
I suggest to not use localized value here as it is part of the logic. The localization for it happens in l345